src/bramble/client/main.js

```diff
@@ -122,7 +122,34 @@ define([
 
     // Expose Filer for Path, Buffer, providers, etc.
     Bramble.Filer = Filer;
-    var _fs = new Filer.FileSystem();
+    var _fs;
+
+    // Set URL param for memory-backed FileSystem
+    var mem_filesystem = "BrambleMemoryFileSystem";
```

Comment:
A few things:

1. In JS, you use `camelCase` to name variables vs. `a_b` style like Python.
2. We probably don't need this variable at all, since you only use it once below.  Just inline it in the `if`
3. Because this will be a user-facing part of the system, let's rename this to `temporaryStorage` (@gideonthomas do you think that's good?  I want something that means "we won't be saving your stuff long.")

Reply:
+1 to the name change